Project.toml

```diff
@@ -18,7 +18,7 @@ DataFrames = "1"
 Lux = "1"
 LuxCUDA = "0.3.3"
 Optimisers = "0.4"
-ProgressMeter = "1"
+ProgressLogging = "0.1.4"
```

Comment:
for compat you wouldnt specify a bugfix release, but rather "0.1", then all potential non-breaking bugfix releases can be included